app/helpers/sort_helper.rb

```diff
@@ -0,0 +1,15 @@
+module SortHelper
+  def table_header_sort_by(heading, path, column:, sort:)
+    order = column == sort.column ? sort.reverse_order : sort.order
+    extra_classes = "sortby--#{order}"
+    extra_classes += " active" if column == sort.column
+
+    govuk_link_to(heading, "#{path}?#{params_with_sort(column, order).to_query}",
```

Comment:
Is the path for sorting always going to be the same as the path the user is currently on? If so it might be easier to just leave the path off entirely and do:

```ruby
govuk_link_to(heading, params_with_sort(column, order), ...)
```

... I think. It's been a long day ☕